extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -474,8 +475,7 @@ export class AmpA4A extends AMP.BaseElement {
           if (!creativeParts || !creativeParts.signature) {
             return Promise.resolve();
           }
-          this.protectedEmitLifecycleEvent_(
-              'adResponseValidateStart', creativeParts);
+          this.protectedEmitLifecycleEvent_('adResponseValidateStart');
```

Comment:
yes I'm thinking we could include the service name which is now available although hopefully in the relatively short term, we will have modified the signature to indicate explicitly which public key should be used @taymonbeal   If you're concerned about multiple, you could modify to only send for the key that passes validation.

I very much want to get this data given the differing results we see for iOS vs Android which I believe may be due to native crypto performance.